src/main/java/org/octri/hpoonfhir/view/ObservationModelBuilder.java

```diff
@@ -0,0 +1,28 @@
+package org.octri.hpoonfhir.view;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.hl7.fhir.dstu3.model.Observation;
+import org.monarchinitiative.fhir2hpo.hpo.HpoConversionResult;
+
+/**
+ * This builder takes care of parsing the observation and the conversion result to construct the view.
+ * 
+ * @author yateam
+ *
+ */
+public class ObservationModelBuilder {
+	
+	public static ObservationModel build(Observation fhirObservation, List<HpoConversionResult> conversionResults) {
+		ObservationModel observationModel = new ObservationModel();
+		observationModel.setDescription(fhirObservation.getCode().getCodingFirstRep().getDisplay());
+		observationModel.setLoincIds(conversionResults.stream().map(it->it.getLoincId().toString()).collect(Collectors.joining(", ")));
+		observationModel.setValue("TODO");
```

Comment:
The way I was setting "value" before was not correct. This model is going to undergo a pretty big redesign, so I just left this as a TODO for now.